lib/rentlinx/version.rb

```diff
@@ -1,3 +1,3 @@
 module Rentlinx
-  VERSION = '0.8.0'
+  VERSION = '0.8.1'
```

Comment:
`master` branch is on 0.8.2 already. I think the best think is to only bump version on `master` branch, though.